app/src/components/RoomAnnouncement.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="roomAnnouncements">
+        <p class="mt-1 common room-ann-text" v-if="roomAnnouncement != null">
+            <i class="far fa-bullhorn"></i> {{roomAnnouncement['msg']}}
+        </p>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: "RoomAnnouncement",
```

Comment:
What's the difference between `RoomAnnouncement` and `Announcement`?

Reply:
Υπάρχει γενική ανακοίνωση και υπάρχει και ανακοίνωση για κάθε αίθουσα ξεχωριστά π.χ. αναβάθμιση CSLab.